packages/twenty-server/src/engine/core-modules/workflow/workflow-trigger.resolver.ts

```diff
@@ -0,0 +1,31 @@
+import { UseGuards } from '@nestjs/common';
+import { Args, Mutation, Resolver } from '@nestjs/graphql';
+
+import { workflowTriggerGraphqlApiExceptionHandler } from 'src/engine/core-modules/workflow/utils/workflow-trigger-graphql-api-exception-handler.util';
+import { WorkflowTriggerService } from 'src/engine/core-modules/workflow/workflow-trigger.service';
+import { Workspace } from 'src/engine/core-modules/workspace/workspace.entity';
+import { AuthWorkspace } from 'src/engine/decorators/auth/auth-workspace.decorator';
+import { JwtAuthGuard } from 'src/engine/guards/jwt.auth.guard';
+
+@UseGuards(JwtAuthGuard)
+@Resolver()
+export class WorkflowTriggerResolver {
+  constructor(
+    private readonly workflowTriggerService: WorkflowTriggerService,
+  ) {}
+
+  @Mutation(() => Boolean)
+  async enableWorkflowTrigger(
```

Comment:
`enableWorkflowTrigger` -> I don't understand the naming here

`triggerWorkflow` ?

Reply:
@martmull It does not really trigger the workflow. It setup the workflow trigger, so it will get triggered later.
I could rename it `setupWorflowTrigger` but this is not that fare from `enableWorkflowTrigger`. So for this one let's remain consistent with `activePieces`? Wdyt?